pkgs/applications/misc/blugon/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub, python3, libX11, libXrandr }:
+
+stdenv.mkDerivation rec {
+  pname = "blugon";
+  version = "1.11.4";
+
+  src = fetchFromGitHub {
+    owner = "jumper149";
+    repo = "blugon";
+    rev = "c53b47eac7f9b23adeb62f8df10f38ffbc0dd156";
+    sha256 = "0x320w2h5nlcgha4345i8ns15akb4kmrdgkh710s4r1n1by4x11r";
+  };
+
+  buildInputs = [ python3 libX11 libXrandr ];
+
+  makeFlags = [ "PREFIX=$(out)" ];
```

Comment:
```suggestion
  installFlags = [ "PREFIX=$(out)" ];
```

Reply:
The `PREFIX` is also required for `make build`.